### ISSUE TITLE
Needs hex/obfuscated checking, also checks for fork bombs

### ISSUE THREAD

**Issue by vxbinaca** [micapucha/Experiments#1]
For when assholes on Ubuntu forums would post fork bombs in hex as solutions to peoples driver issues, they made the software filter for those things.

**Comment by micapucha** [comment]
Yeah, you are right. However, the only practical solution for this is a reverse "jailbreak" (that is, jail the applications so they cannot write outside certain boundaries). That must be implemented at firmware level, like a CFW.

I mean, this script works for the most basic malware (the mai dumps' case), but the payload could be XORed with a random key, generated by calls to multiple functions, be a time bomb... sadly, there is no universal way to detect real malware without executing the binaries in a virtual environment or something like that.
